change/@ni-nimble-components-8eeeed78-306a-426a-a070-ad528135dcf8.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Milan's comment about selection made me try it out in various browsers in Storybook on my machine (macOS 13.5.2). It works well in Safari, Edge, and Chrome but unfortunately Firefox is stopping the selection operation when I drag over the mention view. I can select text or the mention view, but not both.


https://github.com/ni/nimble/assets/10500124/674e7832-f9ae-461d-bd00-b2d442e405db

Reply:
We already came across this issue and I commented on it in [the other comment](https://github.com/ni/nimble/pull/1653#discussion_r1384680808) in detail. This is a known issue in Firefox and here is the link to the issue: https://bug1455893.bmoattachments.org/attachment.cgi?id=9289349

The only workaround to the issue is to have all the text content in the slot instead of constructing ourselves in the template. In that way, the Firefox issue resolves, but creates other issues like selecting using the keyboard when the view node is rendered into the editor. Since this issue is specific only to Firefox we thought of having this issue over other generic issues that are on all other browsers.

Reply:
Oh interesting, didn't realize there was still an issue with Firefox. One thing to try is in normal mode don't have the slot element in the template and then have the slot conditionally when editing-disabled is set (so we can set it display none)

Reply:
Created an issue in Nimble for the issue to track: https://github.com/ni/nimble/issues/1662